src/resources/commandTemplates/templates.ActionCommand.ts

```diff
@@ -247,26 +246,19 @@ async function validateDuration(
       }
     }
 
-  if (
-    ACTION === 'ban' &&
-    ((duration /= 86400000) % 1 !== 0 || duration > 7 || duration < 0)
-  ) {
-    await interaction.followUp({
-      content:
-        'Ban duration must be between 1 and 7 days without hours, minutes, or seconds',
-      ephemeral: true,
-    });
-    return [false, undefined];
-  }
+  if (ACTION === 'ban')
+    return [true, duration / 1000 | 0];
+  // https://stackoverflow.com/questions/7487977/using-bitwise-or-0-to-floor-a-number :trollface:
```

Comment:
this seems to me to be a poor idea for readability's sake- I think you agree too because you added a comment saying what it's doing... Is there any significant advantage you see to doing it this way?

Reply:
oh yeah it was purely for the memes (and totally not because I forgot `math.floor()` exists)